.appends/.github/labels.yml

```diff
@@ -0,0 +1,104 @@
+- name: "bug :bug:"
+  description: "Something isn't working"
+  color: "b60205"
+
+- name: "chore :wrench:"
+  description: "Meta related task such as build, test, linting, maintainers.json etc."
+  color: "000000"
+
+- name: "config change :gear:"
+  description: "Changes to the track or exercise configs"
+  color: "ededed"
+
+- name: "dependencies :arrow_up_small:"
+  description: "Pull requests that update a dependency file"
+  color: "0366d6"
+
+- name: "discussion :speech_balloon:"
+  description: ""
+  color: "fbca04"
+
+- name: "do not merge :construction:"
+  description: "Don't merge until this label is removed"
+  color: "fbca04"
+
+- name: "documentation :book:"
+  description: "Documentation changes"
+  color: "1d76db"
+
+- name: "duplicate :repeat:"
+  description: "This issue or pull request already exists"
+  color: "ffffff"
+
+- name: "enhancement :unicorn:"
+  description: "Changing current behaviour, enhancing what's already there"
+  color: "a2eeef"
+
+- name: "experimental :microscope:"
+  description: "Speculative functionality or implementation"
+  color: "b60205"
+
+- name: "first-timers only :baby:"
+  description: "This issue is specifically for those who are new contributors."
+  color: "b1db51"
+
+- name: "good first issue"
+  description: ""
+  color: "680a9b"
```

Comment:
I believe this should be an org-wide label

Reply:
Yeah, we'll have a specific label for this purpose in the org-wide labels. For now, this is just an initial import that can be cleaned up later.